src/client/app/containers/LineChartContainer.ts

```diff
@@ -18,6 +18,8 @@ function mapStateToProps(state: State) {
 	const timeInterval = state.graph.timeInterval;
 	const unitID = state.graph.selectedUnit;
 	const datasets: any[] = [];
+	const yMinData: number[] = [];
```

Comment:
Since these arrays are per-meter, these need to be defined inside of the for loop by meterID below (see xData and yData arrays). When defined here, these result in a failure to add error bars when more than one meter is graphed.
![image](https://github.com/OpenEnergyDashboard/OED/assets/113216142/d284b241-ffbe-451e-9979-d27bba061a6a)